docs/developers/node-api.md

```diff
@@ -451,7 +451,7 @@ command:
 
 ```sh
 curl -X POST \
-     -H "Authorization: Bearer $AUTH_TOKEN" \
+     -H "Authorization: Bearer $AUTH_TOKEN"
```

Comment:
did you mean to delete this?

Reply:
nope. another i'm not quite sure of..maybe from resolving conflicts